### PR TITLE
torsocks: add page

### PR DESCRIPTION
**Pull request by MIfoodie**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**
- 2.4.0

### CODE REVIEW

File: pages/linux/torsocks.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# torsocks
+
+> Use an external application through the Tor network.
+> More information: <https://github.com/dgoulet/torsocks>.
+
+- Run a command using Tor:
+
+`torsocks {{command}}`
+
+- Toggle your shell in/out of Tor mode:
```
Comment (tricantivu) [inline pages/linux/torsocks.md:10]: Hello, thanks for the contribution. I have the following the suggestion, I believe the description is shorter.

```suggestion
- Enable or disable Tor:
```

**Review by einverne — APPROVED** [review]
Looks good after @tricantivu 's suggestion.

File: pages/linux/torsocks.md (hunk 1 above)
Comment (gutjuri) [reply pages/linux/torsocks.md:10]: > Hello, thanks for the contribution. I have the following the suggestion, I believe the description is shorter.

I think that this doesn't sufficiently convey that the command only toggles TOR mode for *this shell*. I would be fine with the original version.

**Review by gutjuri — APPROVED** [review]

File: pages/linux/torsocks.md (hunk 1 above)
Comment (vitorhcl) [reply pages/linux/torsocks.md:10]:
```suggestion
- Enable or disable Tor in this shell:
```
Considering @gutjuri's cocern, maybe we could go with this?

**Comment by vitorhcl** [comment]
> - **Version of the command being documented (if known):**
> - Latest

@MIfoodie just a minor tip: if possible, write the version itself because it makes it easier to check later if the command changes or something like that.

File: pages/linux/torsocks.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# torsocks
+
+> Use an external application through the Tor network.
```
Comment (vitorhcl) [inline pages/linux/torsocks.md:3]:
```suggestion
> Use any application through the Tor network.
```
"external" application seems a little bit odd to me

**Review by vitorhcl — APPROVED** [review]
Thank you for your contribution! LGTM after we address the open suggestions.

File: pages/linux/torsocks.md (hunk 1 above)
Comment (tricantivu) [reply pages/linux/torsocks.md:10]: I agree

File: pages/linux/torsocks.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+> More information: <https://github.com/dgoulet/torsocks>.
```
Comment (fazlearefin) [inline pages/linux/torsocks.md:4]:
```suggestion
> More information: <https://gitlab.torproject.org/tpo/core/torsocks/>.
```

As it says on the github page, upstream has been moved to https://gitlab.torproject.org/tpo/core/torsocks/ . Also, comparing the commits, gitlab has the newer commits:

https://github.com/dgoulet/torsocks/commits/master/
https://gitlab.torproject.org/tpo/core/torsocks/-/commits/main

File: pages/linux/torsocks.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+> Use any application through the Tor network.
+> More information: <https://gitlab.torproject.org/tpo/core/torsocks/>.
+
+- Run a command using Tor:
+
+`torsocks {{command}}`
+
+- Enable or disable Tor in this shell:
+
+`. torsocks {{on/off}}`
```
Comment (kbdharun) [inline pages/linux/torsocks.md:12]:
```suggestion
`. torsocks {{on|off}}`
```
We generally use this operator to highlight multiple options supported by the placeholder.

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution. I have a minor suggestion for this page, and the PR is GTG.

**Pull request merged**